android/expoview/src/main/AndroidManifest.xml

```diff
@@ -106,6 +106,11 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/provider_paths" />
         </provider>
+
+      <meta-data
+        tools:replace="android:value"
```

Comment:
What would happen without this `tools:replace` property?

Reply:
It won't compile. We need to inform compile that we intentionally provide same meta-data twice.